### DISCUSSION TITLE
How to make a widget for multiple Resources?

### DISCUSSION THREAD

**Discussion by moskoweb** [filamentphp/filament#6681] [Help]
Hello,

I want to create a Widget that can be used in more than one Resource, with 3 standard filters, to count the Total, Active and Inactive, to be displayed in the Resource listing.

The documentation only talks about the Edit and View screens, to get the Model, but in this case, I want access to the Resource, to get the Eloquent collection used for the listing.

<img width="1424" alt="Captura de Tela 2023-06-03 às 10 48 09" src="https://github.com/filamentphp/filament/assets/6052272/7861854d-8a1b-4a21-b713-2a593a9b4962">

Example:
```php
<?php

namespace App\Filament\Widgets;

use Filament\Widgets\StatsOverviewWidget as BaseWidget;
use Filament\Widgets\StatsOverviewWidget\Card;

class ResourceOverview extends BaseWidget
{
    protected static string $resource;
        
    protected function getCards(): array
    {        
        return [
            Card::make('Total', $resource->count())
                ->color('primary'),
                
            Card::make('Active', $resource->where('status', 1)->count())
                ->color('success'),
                
            Card::make('Inactive', $resource->where('status', 0)->count())
                ->color('warning'),
        ];
    }
}
```

**Comment by pboivin** [comment]
Hi @moskoweb , I'm not sure if there's a way to access the table's eloquent collection from a widget. If you extend the `list-records` view on your page, you may be able to pass extra data to your widget with the `:widget-data=""` prop : 
https://github.com/filamentphp/filament/blob/2.x/packages/admin/resources/views/resources/pages/edit-record.blade.php#L2
